pkgs/games/enyo-doom/default.nix

```diff
@@ -20,7 +20,7 @@ mkDerivation rec {
   meta = {
     homepage = "https://gitlab.com/sdcofer70/enyo-doom";
     description = "Frontend for Doom engines";
-    license = stdenv.lib.licenses.gpl2;
+    license = stdenv.lib.licenses.gpl3;
```

Comment:
gpl3only or gpl3plus?

Reply:
Ah, looks like it's gpl3plus judging by the source headers. I'll update this.